server/services/interventionsService.ts

```diff
@@ -501,7 +501,6 @@ export default class InterventionsService {
     })) as Appointment
   }
 
-  /*
   async recordAppointmentAttendance(
```

Comment:
are there existing unit / contract tests for this?

Reply:
The pact tests are a separate ticket. Do you want me to do it in this PR? 

To note: I am merging this into a feature branch rather than `main`.